tools/wake/main.cpp

```diff
@@ -538,7 +538,7 @@ int main(int argc, char **argv) {
   if (debug   && !fd1) fd1 = "debug,info,echo,report,warning,error";
   if (verbose && !fd1) fd1 = "info,echo,report,warning,error";
   if (quiet   && !fd1) fd1 = "error";
-  if (!tty    && !fd1) fd1 = "info,echo,report,warning,error";
+  if (!tty    && !fd1) fd1 = "echo,report,warning,error";
```

Comment:
do the cli flags debug/verbose have effect when running --no-tty?

Reply:
Yeah, they'll be matched first and override this line in specifying output levels.  I think the other things `--no-tty` does (remove spinners, etc.) will still be in effect.

Reply:
Note also that 'tty=false' not just for '--no-tty', but also for just running wake without a tty.

Reply:
Ah, I missed the later `term_init`, thanks!